main.go

```diff
@@ -116,3 +129,50 @@ Arguments:
 		helpTab,
 	)
 }
+
+// This section translates a windows path such as "C:\Users\user" to "/Users/user"
+// and removes the default paths for mingw and git4windows to issues when you
+// specify a file path to start code-server in.
+func gitbashWindowsDir(dir string) string {
+	if dir == "~" { //Special case
+		return "~/"
+	}
+	mountPoints := gitbashMountPointsAndHome()
+
+	// Apply mount points
+	absDir, _ := filepath.Abs(dir)
+	absDir = filepath.ToSlash(absDir)
+	for _, mp := range mountPoints {
+		if strings.HasPrefix(absDir, mp[0]) {
+			resolved := strings.Replace(absDir, mp[0], mp[1], 1)
+			flog.Info("Resolved windows path '%s' to '%s", dir, resolved)
```

Comment:
```suggestion
```

Reply:
Why would you suggest not implementing this? 
A little more detail other than "hey, am suggesting this doesnt go here" would be nice please?

Reply:
Temporarily set to `log.Println` for debugging